sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveQuerySuite.scala

```diff
@@ -857,6 +857,30 @@ class HiveQuerySuite extends HiveComparisonTest with SQLTestUtils with BeforeAnd
     assert(sql(s"list file $testFile").count() == 1)
   }
 
+  test("ADD ARCHIVE command") {
+    val testFile =
+      TestHive.getHiveFile("data/files/archives1/archive1.tar.gz").toURI.toString + "#link1"
+    sql(s"ADD ARCHIVE $testFile")
```

Comment:
I think it would be easier to read if we generate a zip file here. `TestUtils.createJar` creates a jar (zip) file. So It think this utility can be used instead. I added a similar test in `SparkContextSuite.SPARK-33530: basic case for addArchive and listArchives`.

Reply:
Thanks for the advice. It's sounds reasonable.